chapter6.adoc

```diff
@@ -193,17 +230,24 @@ of instructions.
 [width=100%]
 [%header, cols="^3,10,^3, 18, 5"]
 |===
-|Offset|Name           |Size    |Description              | Optional?
-|0     |`capabilities` |8       |<<CAP,  Capabilities >>  | No
-|8     |`control`      |8       |<<CTRL, Control      >>  | No
-|16    |`operand-0`    |8       |<<OP-0, Operand 0    >>  | No
-|24    |`operand-1`    |8       |<<OP-1, Operand 1    >>  | No
-|32    |`operand-2`    |8       |<<OP-2, Operand 2    >>  | Yes
+|Offset|Name           |Size    |Description                 | Optional?
+|0     |`capabilities` |4       |<<CAP, Capabilities     >>  | No
+|4     |`control`      |4       |<<CTL, Control          >>  | No
+|4     |`command`      |4       |<<CMD, Command          >>  | No
+|8     |`status`       |4       |<<STS, Status           >>  | No
```

Comment:
suggest moving STS after CAP, in the above register and in the text description below.

Reply:
done